moped-editor/src/views/dev/LookupsView/index.js

```diff
@@ -26,6 +27,16 @@ const useStyles = makeStyles((theme) => ({
  */
 const createRecordKeyHash = (recordKey) => `#${recordKey.replace("_", "-")}`;
 
+/**
+ * Scroll to a page element based on its key
+ */
+const scrollToTable = (recordKey, refs) => {
+  const ref = refs?.[recordKey];
+  if (ref?.current) {
+    ref.current.scrollIntoView({ behavior: "smooth" });
+  }
+};
+
```

Comment:
To make this feature work, I had to de-couple the scrollTo action from the effect that runs when initially parsing the page route. I think this ends up being cleaner anyway.